release-notes/7.0/known-issues.md

```diff
@@ -100,115 +100,29 @@ More information and workaround can be found at <https://github.com/dotnet/runti
-  ```
-
-2. Locate the `dotnet-user-jwts.deps.json` file and make the following modifications:
-
-```diff
-{
-    "targets": {
-        ".NETCoreApp,Version=v7.0": {
-            "dotnet-user-jwts/7.0.0-rc.1.22426.10": {
-            "dependencies": {
-+               "Microsoft.Extensions.Configuration.Binder": "7.0.0-rc.1.22426.10"
-            },
-+           "Microsoft.Extensions.Configuration.Binder/7.0.0-rc.1.22426.10": {
-+               "dependencies": {
-+                   "Microsoft.Extensions.Configuration.Abstractions": "7.0.0-rc.1.22426.10"
-+               },
-+               "runtime": {
-+                   "lib/net7.0/Microsoft.Extensions.Configuration.Binder.dll": {
-+                       "assemblyVersion": "7.0.0.0",
-+                       "fileVersion": "7.0.22.42610"
-+               }
-+           }
-        },
-    },
-    "libraries": {
-+       "Microsoft.Extensions.Configuration.Binder/7.0.0-rc.1.22426.10": {
-+           "type": "package",
-+           "serviceable": true,
-+           "sha512": "",
-+           "path": "microsoft.extensions.configuration.binder/7.0.0-rc.1.22426.10",
-+           "hashPath": "microsoft.extensions.configuration.binder.7.0.0-rc.1.22426.10.nupkg.sha512"
-+       },
-    }
-}
-```
-
-3. Locate the `Microsoft.Extensions.Configuration.Binder` assembly in the directory associated with the `Microsoft.AspNetCore.App` shared runtime. This will typically be located in a path as follows:
-
-- on Linux/macOS
+  <PropertyGroup>
+    <RuntimeFrameworkVersion>6.0.32</RuntimeFrameworkVersion>
+  </PropertyGroup>
 
-  ```console
-  ~/.dotnet/shared/Microsoft.AspNetCore.App/7.0.0-rc.1.22427.2
-  ```
-
-- on Windows
-
-  ```console
-  C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App\7.0.0-rc.1.22427.2
-  ```
-
-4. Copy the assembly from Step 3 to the `user-jwts` tool directory from Step 1.
-
-- on Linux/macOS
-
-  ```console
-  cp ~/.dotnet/shared/Microsoft.AspNetCore.App/7.0.0-rc.1.22427.2/Microsoft.Extensions.Configuration.Binder.dll ~/.dotnet/sdk/7.0.100-rc.1.22431.12/DotnetTools/dotnet-user-jwts/7.0.0-rc.1.22427.2/tools/net7.0/any
-  ```
-
-- on Windows
-
-  ```console
-  copy C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App\7.0.0-rc.1.22427.2\C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App\7.0.0-rc.1.22427.2 C:\Program Files\ddotnet\sdk\7.0.100-rc.1.22431.12\DotnetTools\dotnet-user-jwts\7.0.0-rc.1.22427.2\tools\net7.0\any
-  ```
-
-5. Note that the install directory for the SDK may not be deleted during uninstall due to applying this workaround, e.g. when updating to 7.0.0-rc.2. If that occurs, delete the directory manually.
-
-This issue will be resolved in .NET 7 RC 2.
+  <ItemGroup>
+    <KnownFrameworkReference  Update="@(KnownFrameworkReference)">
+      <TargetingPackVersion Condition="'%(TargetFramework)' == 'net6.0'">6.0.32</TargetingPackVersion >
+    </KnownFrameworkReference >
+  </ItemGroup>
```

Comment:
Should we log an issue for making this easier to do, maybe via a property we document? Happy to write that up if so.

Reply:
heh, I did already: https://github.com/dotnet/sdk/issues/42196